src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -455,6 +455,12 @@
 			"true",
 			"# If Towny should show players the townboard when they login"
 	),
+	GTOWN_SETTINGS_OUTSIDERS_PREVENT_PVP_TOGGLE(
+			"global_town_settings.outsiders_prevent_pvp_toggle",
+			"true",
+			"# If Towny should allow the toggling of pvp, while others ,who are not part of the town, are in the town",
+			"# If this option is set to false, it may cause a small amount of lag, when /t toggle pvp is run."

```

Comment:
"# When active this feature can cause a bit of lag when the /t toggle pvp command is used, depending on how many players are online."